drivers/w1/w1_ds248x.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2022 Caspar Friedrich <c.s.w.friedrich@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_W1_W1_DS248X_H_
+#define ZEPHYR_DRIVERS_W1_W1_DS248X_H_
+
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/zephyr.h>
+
+#define CMD_1WT	 0x78
```

Comment:
not a blocker but IMO enum is better for these and able to provide editor/compiler warning

Reply:
Also no strong feelings from my side.

Reply:
I would leave it for now and revisit for the follow up driver.